ginga/misc/plugins/Cuts.py

```diff
@@ -12,6 +12,7 @@
 from ginga.misc import Widgets, Plot
 from ginga import GingaPlugin, colors
 from ginga.util.six.moves import map, zip
+from ginga.qtw.QtHelp import SaveDialog
 
```

Comment:
Because this plugin is in `.../ginga/misc/plugins`, it is backend-independent.  This is nice because it avoids having to maintain two separate versions of the plugin.  In order to maintain that, we need to have a generic `SaveDialog` available in the `Widgets`. Just make sure there is one in `ginga.gtkw.Widgets` that works the same.